models/articles.models.js

```diff
@@ -107,6 +107,10 @@ RETURNING *`,
     });
 };
 exports.removeCommentById = (id) => {
+  if (!typeof id === "number") {
```

Comment:
We don't need this block, because the db.query will cause an error if it's looking for something that is not a number from a column defined as an INT/ SERIAL in our database. Then you can handle the bad request error accordingly.